--- ../cache/libpciaccess-0.14/src/linux_sysfs.c	2017-10-23 15:48:29.000000000 +0000
+++ ./src/linux_sysfs.c	2018-02-20 23:15:06.762197373 +0000
@@ -49,8 +49,7 @@
 #include <sys/mman.h>
 #include <dirent.h>
 #include <errno.h>
-
-#if defined(__i386__) || defined(__x86_64__) || defined(__arm__)
+#if  defined(__i386__) || defined(__x86_64__) || defined(__arm__) && !defined(__ANDROID__)
 #include <sys/io.h>
 #else
 #define inb(x) -1
