--- ../rustc-1.44.0-src/src/libstd/sys/unix/net.rs	2020-06-01 10:44:16.000000000 -0500
+++ ./src/libstd/sys/unix/net.rs	2020-06-16 12:59:04.104194566 -0500
@@ -382,7 +382,13 @@
     // If the version fails to parse, we treat it the same as "not glibc".
     if let Some(version) = sys::os::glibc_version() {
         if version < (2, 26) {
-            unsafe { libc::res_init() };
+            #[cfg(not(target_os = "android"))]
+            use libc::res_init;
+            #[cfg(target_os = "android")]
+            extern "C" {
+                fn res_init();
+            }
+            unsafe { res_init() };
         }
     }
 }
